manager/test/api/src/test/resources/test-plans/clients-testPlan.xml

```diff
@@ -67,5 +67,6 @@
   <testGroup name="Deleting Clients (With subordinate elements)">
     <test name="Delete Client 1">test-plan-data/clients/delete/002_delete-client-1.resttest</test>
```

Comment:
We should decide on a consistent start of test numbering

Reply:
Oh, interesting! I didn't notice that they start here with 002. I will rename them all.